public-site/src/pages/index.js

```diff
@@ -37,9 +37,9 @@ const IndexPage = ({ data }) => (
       />
     </SideBySide>
 
-    <p style={{ fontStyle: "italic" }}>NOT: The set is currently sold-out.</p>
+    <p style={{ fontStyle: "italic" }}>NOTE: The set is currently sold-out.</p>
```

Comment:
Good catch, and, just a question: Do you think the all-caps for "NOTE" is janky?